### ISSUE TITLE
New upload format for PAWS SalesForce data

### ISSUE THREAD

**Issue by c-simpson** [CodeForPhilly/paws-data-pipeline#214]
When I upload and execute `Paws Account and Contact Data_12_3_2020.xlsx`, I get the following log message: 
```
paws-compose-server | [2021-03-08 17:39:03,416] INFO in file_uploader: Start uploading file: Paws Account and Contact Data_12_3_2020.xlsx
paws-compose-server | [2021-03-08 17:39:38,482] ERROR in file_uploader: No sources found in upload
```
And it does no further processing.

The `Salesforce_PAWS Contacts and Accounts.xlsx` file dated 2020-06-25 gets processed without any trouble.

The files seem to be same out through column O, but then there are differences:

 Col|2020-12-03|2020-06-25
---|---|---
P|Account ID 18|Account ID
Q|Volgistics ID|   
R|Person ID|   

Is this an intake issue or a problem when exporting the data?

**Comment by kfettich** [comment]
This is an intake issue; the 6/25 file had an "Account ID" field that was truncated, and this was updated with the Account ID 18 from the newer file; additionally, the Volgistics ID and the Person ID are the manually mathed IDs that Lauren is saving in Salesforce (which she only started in the 2nd part of last year)

**Comment by c-simpson** [comment]
Just to clarify, should we be modifying the intake process to use the 2020-12-03 file as the standard going forward?

**Comment by c-simpson** [comment]
Closed by [af80ee7](https://github.com/CodeForPhilly/paws-data-pipeline/commit/af80ee79a229b28d95d99886904ce3382aa2f955)
